easybuild/easyconfigs/l/libgeotiff/libgeotiff-1.5.1-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'libgeotiff'
+version = '1.5.1'
+
+homepage = 'https://directory.fsf.org/wiki/Libgeotiff'
+description = """Library for reading and writing coordinate system information from/to GeoTIFF files"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['https://github.com/OSGeo/libgeotiff/archive']
+sources = [{
+    'download_filename': '%(version)s.tar.gz',
+    'filename': SOURCE_TAR_GZ
+}]
+checksums = ['fb04491572afb25ffe60239fdfdcfa2c64e6cf644cad9b0b922b10115ccbd488']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+    ('libtool', '2.4.6'),
+    ('Automake', '1.16.1'),
+    ('Autoconf', '2.69'),
```

Comment:
Except for `binutils`, none of these are needed if the prepared source tarball is used instead